additional_tasks/task_11.py

```diff
@@ -0,0 +1,13 @@
+
+def show_max_num(*args: int):
```

Comment:
Тут невірна анотація типів.
Знову ж таки у поточній функції не має необхідності, оскільки набагато легше викликати `max` на місці.

```suggestion
def show_max_num(*args: Tuple[int, ...]) -> int:
```